tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -355,4 +355,82 @@ public function theFollowingMimeTypesShouldExistForUser(string $shouldOrNot, Tab
 			}
 		}
 	}
+
+	/**
+	 * @Then the app list response should contain the following template information for office :app:
+	 *
+	 * @param string $app
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function theAppListResponseShouldContainTheFollowingTemplateInformationForOffice(string $app, TableNode $table): void {
+		$responseArray = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse());
+
+		if (!isset($responseArray['mime-types'])) {
+			Assert::fail("Expected 'mime-types' in the response but not found.\n" . print_r($responseArray, true));
+		}
```

Comment:
this would good
```suggestion
		Assert::assertArrayHasKey("mime-types", $responseArray, "Expected 'mime-types' in the response but not found.\n" . print_r($responseArray, true));
```